common/libs/framing/soft_rx_buffer.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __SOFT_RX_BUFFER_H
```

Comment:
The include guard should be SOFT_RX_BUFFER_H_

Reply:
Please address this and other previous comments in this file.

Reply:
Done